app/utilities/accounting.py

```diff
@@ -17,6 +17,8 @@ async def get_account(email: str):
 
 async def create_account(email: str, password: str):
 
+    # TODO: Find one and insert in one atomic operation!!!
```

Comment:
Have a look at `update()` with `upsert` set to `True`. Maybe this is what you mean.